components/observations/ObservationsFilterForm.tsx

```diff
@@ -323,12 +323,13 @@ export const ObservationsFilterForm: React.FunctionComponent<ObservationsFilterF
                     </View>
                   )}
                   {mapLayer && (
+                    // TODO: Render in two groups with respective labels: <Center_ID> Forecast Zones & Other Regions
                     <CheckboxSelectField
                       name="zones"
                       items={
                         initialFilterConfig.zones.length > 0
                           ? initialFilterConfig.zones.map(z => ({label: z, value: z}))
-                          : mapLayer.features.map(feature => ({label: feature.properties.name, value: feature.properties.name}))
+                          : [...mapLayer.features.map(feature => ({label: feature.properties.name, value: feature.properties.name}))]
```

Comment:
Why do we need the array destructuring here?

Reply:
Great question! This is a holdover from an earlier attempt of passing the alternate zones separately, and spreading the two arrays here.